Async-Simulator/src/cliUI/LoadingBar.js

```diff
@@ -0,0 +1,29 @@
+const process = require("process")
+const rdl = require("readline")
+class LoadingBar {
+    constructor(size) {
+        this.size = size
+        this.cursor = 0
+        this.timer = null
+    }
+    start() {
+
+        process.stdout.write("\x1B[?25l")
+        for (let i = 0; i < this.size; i++) {
+              process.stdout.write("\u2591")
+
```

Comment:
Are these spaces necessary or maybe because you still wanna add somethings there later?

Reply:
they are dots that aim to make it feel like a bar filling. https://www.fileformat.info/info/unicode/char/2591/index.htm